pkg/clients/clients.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 var instances struct {
-	Etcd       *etcd.Client
+	Etcd       etcd.Client
```

Comment:
In order to call GetVersion, you need the client interface object and not a pointer.